data/state/accountsDB.go

```diff
@@ -98,6 +102,32 @@ func (adb *AccountsDB) addCodeToTrieIfMissing(codeHash []byte, code []byte) erro
 	return nil
 }
 
+// ClosePersister will close trie persister
+func (adb *AccountsDB) ClosePersister() error {
+	closedSuccessfully := true
+
+	err := adb.mainTrie.ClosePersister()
+	log.LogIfError(err)
```

Comment:
you are anyway checking for error != nil, so I wouldn't do logIfError as that would mean you are doing the check twice.
you can directly log it inside if err!= nil{}

Reply:
done.